Nighthold/Aluriel.lua

```diff
@@ -112,7 +114,9 @@ end
 
 function mod:OnEngage()
 	annihilateCount = 1
+	wipe(frostbittenStacks)
 	self:Bar(212492, timers[212492][annihilateCount]) -- Annihilate
+	self:OpenInfo(212647, self:SpellName(212647))

```

Comment:
Should only be shown when players have stacks instead of during the whole fight.